recipes/opensim/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+
+# TODO: Dependencies should be separate packages...
+mkdir opensim_dependencies_build
+cd opensim_dependencies_build
+cmake ../dependencies/ \
+      -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+      -DCMAKE_BUILD_TYPE=Release
+make -j8
```

Comment:
No `make install` ?

Reply:
Correct. This "dependencies" project uses a CMake "superbuild" (ExternalProject_Add), which installs the projects with `make`; there is no `install` target.

Reply:
Ah, thanks. Since there are conda-forge packages for `simbody` and `docopt`, so they should be used, unless it's impossible.

Reply:
Ah right; I forgot that @moorepants finished creating the `simbody` package. That should be possible! We'll need to update the version for the simbody package.